pkg/apis/ark/v1/register.go

```diff
@@ -69,6 +69,7 @@ func CustomResources() map[string]typeInfo {
 		"ResticRepository":       newTypeInfo("resticrepositories", &ResticRepository{}, &ResticRepositoryList{}),
 		"BackupStorageLocation":  newTypeInfo("backupstoragelocations", &BackupStorageLocation{}, &BackupStorageLocationList{}),
 		"VolumeSnapshotLocation": newTypeInfo("volumesnapshotlocations", &VolumeSnapshotLocation{}, &VolumeSnapshotLocationList{}),
+		"ServerStatusRequest":    newTypeInfo("statusrequests", &ServerStatusRequest{}, &ServerStatusRequestList{}),
```

Comment:
Does it need to be this?

```suggestion
		"ServerStatusRequest":    newTypeInfo("serverstatusrequests", &ServerStatusRequest{}, &ServerStatusRequestList{}),
```